### ISSUE TITLE
Check if the "lime" command is installed

### ISSUE THREAD

**Issue by Gama11** [openfl/lime-vscode-extension#11]
Right now, the extension just fails silently if that's not the case. It should show a helpful error to the user so he knows what's going on.

**Comment by ibilon** [comment]
Just had that issue
```
> Executing task: lime build neko <
/bin/bash: lime: command not found
The terminal process terminated with exit code: 127
```

I only have a bash alias for convenience but not the command.
Lime (and all haxelibs) is installed in my home directory so adding a global executable doesn't make sense.

Could the extension run `haxelib run lime` instead? It's a lot more likely to succeed.

**Comment by Gama11** [comment]
Closed by #36. You can now add this to your settings:

```json
"lime.command": "haxelib run lime"
```
